code/datums/mob_descriptors/descriptors/other.dm

```diff
@@ -18,12 +18,12 @@
 	if(!isliving(user))
 		return TRUE
 	var/mob/living/living_user = user
-	// Other defiants can not see
-	if(living_user.defiant)
+	// Other defiants can not see - Commented out for now so people don't ZAPE
+	/*if(living_user.defiant)
 		return FALSE
-	// Further than 2 tiles dont see it
+	// Further than 2 tiles dont see it - Commented out for now to always know who you can't ZAPE
 	if(get_dist(described, living_user) > 2)
-		return FALSE
+		return FALSE*/
```

Comment:
I think we should just remove this check. And the above too, honestly.

Reply:
Don't comment these out, just remove them. The codebase is littered with commented out lines that nobody cares about. If you don't this will still remain commented out a year from now, propbably.